usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -73,16 +73,20 @@ def translateSambaToIpp(networkPath):
 # --------------------
 
 def _installPrinter(username, networkPath, name):
-    logging.debug("Installing Printer {0} on {1}".format(name, networkPath))
-    installCommand = ["lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
+    logging.info("Install Printer {0} on {1}".format(name, networkPath))
+    installCommand = ["timeout", "10", "lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
     logging.debug("* running '{}'".format(" ".join(installCommand)))
-
-    if not subprocess.call(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0:
+    p = subprocess.Popen(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, error = p.communicate()
+    if p.returncode  == 0:
+        logging.debug("* Success Install Printer!")
+        return True
+    elif p.returncode == 124:
+        logging.debug("* Error timeout lpadmin")
```

Comment:
```suggestion
        logging.fatal(f"* Timeout error while installing printer {name} on {networkPath}")
```
The log should be fatal and contain some more information.